rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1074,6 +1063,19 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		for (int i = 0; i < ui->lleList->count(); i++)
 		{
 			items.push_back(ui->lleList->item(i)->clone());
+
+			if (id >= 0)
+			{
+				const std::string text = sstr(items[i]->text());
+
+				// Select default LLE sprx list for manual lib loading, otherwise all libs are unchecked and hidden
+				const bool lle_default = g_prx_list.count(text) && !g_prx_list.at(text);
+
```

Comment:
Mismatch between comment and code...
The comment suggests:
const book selected = lle_default && id == static_cast<int>(lib_loading_type::manual);
items[i]->setCheckState(selected ? Qt::Checked : Qt::Unchecked);

Reply:
Solved comments.

Reply:
It still doesn't make sense since you don't really uncheck them